network/src/peers/peer/peer.rs

```diff
@@ -157,14 +157,23 @@ impl Peer {
                     if data.is_none() {
                         break;
                     }
-                    let data = match data.unwrap() {
+                    let  data = match data.unwrap() {
```

Comment:
```suggestion
                    let data = match data.unwrap() {
```

Reply:
Good catch, for some reasone `rustfmt` is being funky in that long `select!` body.